app/models/subscription.rb

```diff
@@ -7,4 +7,6 @@ class Subscription < ActiveRecord::Base
   validates :email,
             :user,
             :position, presence: true
+
+  scope :notification, -> { where(notification: true) }
```

Comment:
Scope names should generally be an adjective or some kind of modifying prepositional phrase.